.yarnrc.yml

```diff
@@ -5,6 +5,8 @@ enableGlobalCache: true
 logFilters:
   - code: YN0013
     level: discard
+  - code: YN0062 # Incompatible platform warnings. (e.g. fsevents)
+    level: discard
```

Comment:
Can we target the specific messages instead of the error code? i'm concerned that hidden the whole error code can hide future problems if the list of incompatible packages goes up.

Reply:
done in 6cbbdb2